backend/src/v5/models/tickets.js

```diff
@@ -67,12 +71,14 @@ Tickets.updateTicket = async (teamspace, project, model, oldTicket, updateData,
 
 	await DbHandler.updateOne(teamspace, TICKETS_COL, { _id: oldTicket._id }, { $set: toUpdate, $unset: toUnset });
 
-	publish(events.MODEL_TICKET_UPDATE, { teamspace,
+	publish(events.MODEL_UPDATE_TICKET, { teamspace,
 		project,
 		model,
 		ticket: oldTicket._id,
 		author,
+		isFederation,
 		changes,
+		updateData,
```

Comment:
don't send `updateData`, I think changes is sufficient?

The event listender can extrapolate what is in `updateData` base on `changes `